src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -238,6 +239,73 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
                         Slots.QUESTION_ADDITIONAL_INFO, additionalInfo);
     }
 
+    /**
+     * Constructs results statistics for each student.
+     * Statistics to student is only shown when visibility setting is permissive enough
+     * i.e., if the ranks for all team members are in the results bundle.
+     */
+    private String getStudentQuestionResultsStatisticsHtml(
+            String studentEmail, FeedbackQuestionAttributes question, FeedbackSessionResultsBundle bundle) {
+        if (question.recipientType.isTeam()
+                    && (!question.showResponsesTo.contains(FeedbackParticipantType.RECEIVER)
+                                || !question.showResponsesTo.contains(FeedbackParticipantType.STUDENTS))) {
```

Comment:
add comment
`// at least should be able to viewed by other students for team recipient`